code/game/objects/items/crayons.dm

```diff
@@ -427,7 +427,7 @@
 				if(C.reagents)
 					C.reagents.add_reagent("welding_fuel", 5)
 					C.reagents.add_reagent("ethanol", 5)
```

Comment:
Check pocket sand or paper airplanes throw_impact, both have checks for eye protection that you can probably just copy and paste